plugins/pencilblue/templates/angular/admin/content/articles/article_form.html

```diff
@@ -35,6 +36,21 @@
 			});
 		};
 
+		$scope.generateUrl = function() {
+			$scope.article.url = $scope.article.headline.replace(/\s+/g, '-').toLowerCase();
```

Comment:
This can be it's own function that returns the generated URL, so we don't need to have the regex defined twice.